### ISSUE TITLE
Écrire code basé sur example de classifier pytorch (MNIST)

### ISSUE THREAD

**Issue by sofian** [a3lab/crocodile#14]

**Comment by sofian** [comment]
I guess as a first step I would try to change the model to use Conv1d instead of Conv2d: https://pytorch.org/docs/master/generated/torch.nn.Conv1d.html

**Comment by sofian** [comment]
Il y a pas de Dropout2d il faut simplement utiliser Dropout à la place.

**Comment by sofian** [comment]
Il existe max_pool1d: https://www.tensorflow.org/api_docs/python/tf/nn/max_pool1d
